app/lambda/get_transcription_comprehend/lambda_function.py

```diff
@@ -0,0 +1,71 @@
+import json
+import sys
+
+sys.path.append("../")
+import boto3
+from lambda_base import LambdaBase
+from constant_variables import *
+
+class GetTranscriptionComprehendLambda(LambdaBase):
```

Comment:
Can we rename this to "GetSessionDataLambda"?

Currently this lambda gets data related to only Comprehend and Transcription, but we may add in other data also. Keeping that in mind ,GetSessionDataLambda might be a better name?

Reply:
Done.